src/suite/types.ts

```diff
@@ -45,6 +45,9 @@ export type ApplicationArgs = {
   podAnnotations?: Input<{ [key: string]: Input<string> }>;
   ports?: k8s.types.input.core.v1.ContainerPort[];
   servicePorts?: k8s.types.input.core.v1.ServicePort[];
+  backendConfig?: Input<{
+    customResponseHeaders?: Input<string[]>;
```

Comment:
Just wondering if this can be dynamic somehow? Since cors headers don't allow returning multiple origins only specific or `*`.

Reply:
Not possible to be dynamic. Though this is only in production where we have this, so the value would be `api.daily.dev`, no?

Though, it is probably better to add it to each service

Reply:
Our current CORS setting is more advanced than this afaik.
Usually you have a list of rules and if the domain is valid you return only this specific domain in the header. This implementation will expose your rules which is not a best practice. So I'm not sure we should proceed with this approach

Reply:
Yeah thats what I thought as well. Maybe better to add CORS handling to those services directly?

Reply:
Yeah, probably so.

Though we can still add this, as we can add any other header we want if we need to